src/WebExplore/Explore.tsx

```diff
@@ -61,19 +61,15 @@ class Explore extends React.Component<ExploreProps, any> {
     const { navigation } = this.props
     const { nativeEvent = {} } = event
     const { data = '{}' } = nativeEvent
-    let pathname = '/'
     try {
       const eventMessage = JSON.parse(data)
       const urlData = eventMessage.json
       if (eventMessage.type === 'navigation') {
-        pathname = urlData.pathname || '/'
+        if ( urlData.search ==='?purchase_successful=true') {
```

Comment:
What happens if there are utm parameters or some other garbage? Would doing something like this work?
```suggestion
        if ( urlData.search.indexOf('purchase_successful=true') > -1) {
```

Reply:
This also future proofs in case we want to pass more props back through the url in the future.

Reply:
I used includes because it returns a boolean value.